libs/constants/src/maps/incompatible-gamemodes.map.ts

```diff
@@ -0,0 +1,12 @@
+import { Gamemode as G } from '../';
+
+export const IncompatibleGamemodes: ReadonlyMap<G, G[]> = new Map([
```

Comment:
Is this used anywhere yet? I'm not sure how to interpret this structure.

Reply:
Yes, this is used when generating leaderboards during map submission. The idea is to generate leaderboards for every mode the submitter suggests that not *certainly* incompatible. So surf comes up a lot since any non-surf map you could maybe play in surf should just be played in bhop instead.

Reply:
See `getCompatibleLeaderboards` in leaderboardHandler.util for usage